### ISSUE TITLE
current master c6e380c does not compile with VS2013

### ISSUE THREAD

**Issue by fstorck** [beto-rodriguez/Live-Charts#446]
When trying to build the project with VS2013 I get the following error:

Error	3	Friend access was granted to 'LiveCharts.Wpf,PublicKey=0024000004800000940000000602000000240000525341310004000001000100fd75e78956f97c7369ca28eb4214633471da9d13357dc192414bec59b911c99718d9d16a803e527a40a42f947bb297e9d2236d9e541937c4acfc0bc1ca8ee57114963445259b14532d56951de47e773d790110f63b04721bb853c23d9db25c45b83b45dea9aa4157aba95ad890bf69a7d2583e711c786a32817ab1937f43e7d5', but the output assembly is named 'LiveCharts.Wpf, Version=0.9.0.0, Culture=neutral, PublicKeyToken=null'. Try adding a reference to 'LiveCharts.Wpf,PublicKey=0024000004800000940000000602000000240000525341310004000001000100fd75e78956f97c7369ca28eb4214633471da9d13357dc192414bec59b911c99718d9d16a803e527a40a42f947bb297e9d2236d9e541937c4acfc0bc1ca8ee57114963445259b14532d56951de47e773d790110f63b04721bb853c23d9db25c45b83b45dea9aa4157aba95ad890bf69a7d2583e711c786a32817ab1937f43e7d5' or changing the output assembly name to match.	c:\_Dev\github\Live-Charts\Core40\bin\Debug\LiveCharts.dll	WpfView

Tried to using this commit : [https://github.com/beto-rodriguez/Live-Charts/commit/c6e380cf1b52359b007074cad861d2045a3de7aa](url)

**Comment by rusanov-vladimir** [comment]
You are trying to build .net core project with VS2013, that doesn't support .net core out of box.

**Comment by fstorck** [comment]
Hi Vladimir,

 

ok, but it looks that VS2013 can be used, but it is a bit difficult to integrate .net core support. I would suggest it should be mentioned that VS2015 is recommended for building or is there an easier solution for supporting VS2013.

 

Best regards,

                              Florian



Gesendet: Mittwoch, 25. Januar 2017 um 13:41 Uhr
Von: "Vladimir Rusanov" <notifications@github.com>
An: beto-rodriguez/Live-Charts <Live-Charts@noreply.github.com>
Cc: fstorck <github@florianstorck.de>, Author <author@noreply.github.com>
Betreff: Re: [beto-rodriguez/Live-Charts] current master c6e380c does not compile with VS2013 (#446)


You are trying to build .net core project with VS2013, that doesn't support .net core out of box.

**Comment by rusanov-vladimir** [comment]
@fstorck  Hi,
I've wrote the comment because I've noticed in path *Core40* and associated it with .NET Core. On the second thought I don't remember .NET Core to be used in this repo, so I've deleted the comment. To be honest I don't have VS2013 installed so I cannot repro your problem. I think it's better to let someone with VS2013 to check your problem.

Regards,

**Comment by beto-rodriguez** [comment]
@fstorck This is normally happens when I mess up and while working to publish the nuget packages I commit a reference to this assemblies.

The assemblies that I published in Nuget are signed by me:

     >LiveCharts.Wpf,PublicKey=---my local sign--

The sign is not included in the repository, in only exist in my local machine, that is why you got this error.

I am sorry for this, it does not happens often, I verified and the current repo references the correct assemblies.

I will close this for now, if you face any other issue just let me know.
